components/Avatar.html

```diff
@@ -36,7 +36,91 @@
+                        <span class="comp__detail-text">.avatar-lg-size, .avatar-md-size, .avatar-sm-size, .avatar-xs-size.</span>  
+                    </p> 
+                    <div class="component__view avatar__comp">
+                        <div class="avatar avatar-lg-size avatar-item">
+                            <img src="../assets/images/image3.jpg" alt="avatar" class="avatar_image"/>
+                        </div>
+                        <div class="avatar avatar-md-size avatar-item">
+                            <img src="../assets/images/image2.jpg" alt="avatar" class="avatar_image"/>
+                        </div>
+                        <div class="avatar avatar-sm-size avatar-item">
+                            <img src="../assets/images/image1.jpg" alt="avatar" class="avatar_image"/>
+                        </div>
+                        <div class="avatar avatar-xs-size avatar-item">
+                            <img src="../assets/images/image4.jpg" alt="avatar" class="avatar_image"/>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/qYxOzvGiwpRWpgFZnTSJ"
+                        style="width: 732px; height: 402px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+
+                    <p class="component__heading-primary">Text Avatar</p>
+                    <p class="component__sub-text">
+                        Avatars containing simple characters can be created by passing a paragranph tag with class <span class="comp__detail-text">.lg__text-avatar, .md__text-avatar, .sm__text-avatar, .xs__text-avatar</span>. for different font sizes.
+                    </p> 
+                    <div class="component__view avatar__comp">
+                        <div class="avatar avatar-lg-size avatar-item primary-avatar">
+                            <p class="lg__text-avatar">AS</p>
+                        </div>
+                        <div class="avatar avatar-md-size avatar-item primary-avatar">
+                            <p class="md__text-avatar">AS</p>
+                        </div>
+                        <div class="avatar avatar-sm-size avatar-item primary-avatar">
+                            <p class="sm__text-avatar">AS</p>
+                        </div>
+                        <div class="avatar avatar-xs-size avatar-item primary-avatar">
+                            <p class="xs__text-avatar">AS</p>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/2NWJU2SfYl1VRREjpqVz"
+                        style="width: 833px; height: 402px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid inline styles

Reply:
Kuldeep that's the code snippet link i'm copying from Carbon so this inline style is included in that